src/main/java/seedu/address/model/person/TaskDescription.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.model.person;
```

Comment:
Perhaps it would be better to remove this class, since there is already a `Task` class. I presume it's just a small oversight.